src/views/SpatialDisplay.vue

```diff
@@ -47,10 +48,15 @@ import { ColourMap } from '@deltares/fews-web-oc-charts'
 import ColourBar from '@/components/ColourBar.vue'
 import { Layer } from '@deltares/fews-wms-requests'
 import {LayerGroup} from "@deltares/fews-wms-requests/src/response/getCapabilitiesResponse";
-
-interface MapboxLayerOptions {
-  name: string;
-  time: Date;
+import { toWgs84 } from '@turf/projection';
+import { point } from '@turf/helpers';
+
+interface BoundingBox {
```

Comment:
Can we use the `BoundingBox` interface from [@deltares/fews-wms-request](https://github.com/Deltares/fews-wms-requests/blob/e2a8c0134ad47963a347cde660852ba438e99abe/src/response/getCapabilitiesResponse.ts#L70)?